pkg/collector/corechecks/system/disk/disk/disk.go

```diff
@@ -2,138 +2,252 @@
 // under the Apache License Version 2.0.
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
-//go:build !windows
 
-//nolint:revive // TODO(PLINT) Fix revive linter
+//nolint:revive // TODO(AGENTRUN) Fix revive linter
```

Comment:
Better to remove the `//nolint:revive` in `pkg/collector/corechecks/system/disk/disk/...` files and add a Package comment 😄

Reply:
In fact, I'm gonna fix revive errors and remove this